### PR TITLE
Aic 1 create chevron variants

### PR DESCRIPTION
**Pull request by malexa21**
I am having problems with introducing in Storybook.

### CODE REVIEW

File: src/shared/components/atoms/ChevronDown/ChevronDown.stories.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
 import type { Meta, StoryObj } from "@storybook/react";
-import { ChevronDown } from "./ChevronDown";
+import { ChevronDownVariant } from "./ChevronDown";
 
 const meta = {
   title: "atoms/ChevronDown",
-  component: ChevronDown,
+  component: ChevronDownVariant,
```
Comment (CezarMocanu-aily) [inline src/shared/components/atoms/ChevronDown/ChevronDown.stories.ts]: The component should be `ChevronDown`. A variant is a type of that component. Please test the storybook by running 
```
npm run storybook
```

You should be able to go trough all the chevron types

File: src/shared/components/atoms/ChevronDown/ChevronDown.stories.ts [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
   parameters: {
     layout: "centered",
   },
   tags: ["autodocs"],
   argTypes: {},
   args: {},
-} satisfies Meta<typeof ChevronDown>;
+} satisfies Meta<typeof ChevronDownVariant>;
```
Comment (CezarMocanu-aily) [inline src/shared/components/atoms/ChevronDown/ChevronDown.stories.ts]: type should be ChevronDown

File: src/shared/components/atoms/ChevronDown/ChevronDown.stories.ts [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
 export default meta;
 type Story = StoryObj<typeof meta>;
 
-export const Default: Story = {};
+export const ChevronDown: Story = {};
```
Comment (CezarMocanu-aily) [inline src/shared/components/atoms/ChevronDown/ChevronDown.stories.ts]: Here you need to export one for each variant so you should export 

```
export const Outlined: Story = {
    args: {
       variant: ChevronDownVariant.outline,
    }
}

export const Solid: Story = {...}
export const Mini: Story = {...}
export const Micro: Story = {...}
```

**Review by CezarMocanu-aily — CHANGES_REQUESTED** [review]

**Review by cezarmocanu — CHANGES_REQUESTED** [review]
Please fix comments

**Review by cezarmocanu — APPROVED** [review]

**Pull request merged**